src/Components/Samples/BlazorServerApp/App.razor

```diff
@@ -1,4 +1,5 @@
-﻿<Router AppAssembly="@typeof(Program).Assembly">
+﻿@implements IRazorComponentApplication<App>
```

Comment:
Is this required? Assuming it's not (for back-compat) then maybe we could not have this here, and only have it on the United sample.

Reply:
Ah, not sure why this ended up here. I'll remove it.

Reply:
```suggestion
```